### DISCUSSION TITLE
How can I skip the 2FA after stolen my phone and laptop?

### DISCUSSION THREAD

**Discussion by zinko1998** [community/community#129786] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Last week, my phone and laptop has been stolen and I can't access any 2FA options but I can prove that is my account with my primary and secondary email or phone number or my Passport. Is there any way to get my account back ?

**Comment by LilWhiteFoxxx** [comment]
You need to contact the support department of email

**Comment by LilWhiteFoxxx** [comment]
You need to go to the phone network branches

**Comment by LiteBrite82** [comment]
Hi @zinko1998 ! 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
